src/irr/CMakeLists.txt

```diff
@@ -0,0 +1,513 @@
+
```

Comment:
wtf... there's no such file anymore!? this is `irr` not `nbl` how did you even get this?